### PR TITLE
Move GraphConvPredictor to Library

### PR DESCRIPTION
**Pull request by natsukium**
I moved GraphConvPredictor class which is used in examples to library.

### CODE REVIEW

File: chainer_chemistry/models/prediction/predictor.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        # type: (numpy.ndarray, numpy.ndarray) -> chainer.Variable
+        x = self.graph_conv(atoms, adjs)
+        if self.mlp:
+            x = self.mlp(x)
+        return x
+
+    def predict(self, atoms, adjs):
+        # type: (numpy.ndarray, numpy.ndarray) -> chainer.Variable
+        with chainer.no_backprop_mode(), chainer.using_config('train', False):
+            x = self.forward(atoms, adjs)
```
[the remark this answers is not in the document]
Comment (natsukium) [reply chainer_chemistry/models/prediction/predictor.py]: In chainer v4, forward method is not supported.

https://docs.chainer.org/en/v4.0.0/guides/models.html

File: chainer_chemistry/models/prediction/predictor.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# from typing import Optional  # NOQA
```
[the remark this answers is not in the document]
Comment (natsukium) [reply chainer_chemistry/models/prediction/predictor.py]: Can I make another PR for typing support?
It might just add typing module to `install_requires`.

**Comment by natsukium** [comment]
Fix some problems based on comments.

File: chainer_chemistry/models/prediction/graph_conv_predictor.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+class GraphConvPredictor(chainer.Chain):
+    """Wrapper class that combines a graph convolution and MLP."""
+
+    def __init__(
+            self,
+            graph_conv,  # type: chainer.Link
+            mlp=None,  # type: Optional[chainer.Link]
+            label_scaler=None,  # type: Optional[chainer.Link]
+            postprocess_fn=None  # type: Optional[chainer.FunctionNode]
```
Comment (corochann) [inline chainer_chemistry/models/prediction/graph_conv_predictor.py]: I expect `callable`

**Comment by corochann** [comment]
LGTM

**Pull request merged**